src/main/java/hello/HelloController.java

```diff
@@ -36,4 +37,16 @@ public String index() {
     public String getUser() {
         return "John Doe";
     }
+    
+    public Object clone() {
+        return null;
+    }
+
+    public int compareTo(Object value) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override "equals(Object obj)" to comply with the contract of the "compareTo(T o)" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.api.cicd.siriuscloudservices.com/coding_rules#rule_key=squid%3AS1210)